draft-ietf-gnap-core-protocol.md

```diff
@@ -99,6 +99,13 @@ informative:
                 ins: Å. Axeland
             -
                 ins: O. Oueidat
+    HELMSCHMIDT2021:
```

Comment:
```suggestion
    HELMSCHMIDT2022:
```

Reply:
I'd just put this in as a placeholder until you can publish it, we'll obviously add all the information when we can. :)